src/group/mls_group/api.rs

```diff
@@ -72,7 +72,7 @@ pub trait Api: Sized {
         &mut self,
         mls_plaintext: MLSPlaintext,
         proposals: Vec<MLSPlaintext>,
-        own_key_packages: Vec<KeyPackageBundle>,
+        own_key_packages: &[KeyPackageBundle],
```

Comment:
This still passes the KeyPackageBundles by value. I think what you intended was

`&[&KeyPackageBundle]`

Reply:
Isn't a slice `&[KeyPackageBundle]` already a reference and analogous to `&Vec<KeyPackageBundle>? My intention was for the ownership of the `KeyPackageBundle`s not to change when the function is called. I thought using slices here would do the trick?

Reply:
https://doc.rust-lang.org/book/ch04-03-slices.html

Reply:
@franziskuskiefer so slices are inherently references and we're fine here? I re-read the section on slices and that's how I understand it.

Reply:
Yeah, unless @raphaelrobert sees a reason to have references of references here.

Reply:
You're right, apologies.